examples/network_delay.yaml

```diff
@@ -9,6 +9,7 @@ metadata:
   name: network-delay
   namespace: chaos-demo
 spec:
+  specifiedEnv: lima
```

Comment:
oh, we have a different key in spec. Maybe just use the same for consistency?

Reply:
We have a different variable internally, but the json tag matches. But I'll be changing both to `environment`